libvcx/src/aries/handlers/connection/connection.rs

```diff
@@ -41,6 +41,13 @@ struct ConnectionInfo {
     their: Option<SideConnectionInfo>,
 }
 
+#[derive(Debug, PartialEq)]
+pub enum ConnectionState {
```

Comment:
We'll be able to remove this enum once we split general `Connection` into `Inviter` and `Invitee` VCX objects, analogically to `issuer` and `holder`